tutorials/plot_receptive_field.py

```diff
@@ -0,0 +1,227 @@
+# Lags will now be in axis 1, then we reshape to vectorize
+X_del = delay_time_series(X, tmin, tmax, sfreq, newaxis=1, axis=-1)
+X_del = X_del.reshape([n_epochs, -1, n_times])
+n_features = X_del.shape[1]
+weights_sim = weights.ravel()
+
+# Simulate a neural response to the sound, given this STRF
+y = np.zeros([n_epochs, n_times])
+for ii, iep in enumerate(X_del):
+    # Simulate this epoch and add random noise
+    noise_amp = .0005
+    y[ii] = np.dot(weights_sim, iep) + noise_amp * rng.randn(n_times)
+
+# Plot the first 2 trials of audio and the simulated electrode activity
+X_plt = scale(np.hstack(X[:2]).T).T
+y_plt = scale(np.hstack(y[:2]))
+time = np.arange(X_plt.shape[-1]) / sfreq
+fig, axs = plt.subplots(2, 1, figsize=(10, 10), sharex=True)
+axs[0].pcolormesh(time, freqs, X_plt, vmin=0, vmax=4, cmap='viridis')
+axs[0].set_title('Input auditory features')
+axs[0].set(ylim=[freqs.min(), freqs.max()], ylabel='Frequency (Hz)')
+axs[1].plot(time, y_plt)
+axs[1].set(xlim=[time.min(), time.max()], title='Simulated response',
+           xlabel='Time (s)', ylabel='Activity (a.u.)')
+mne.viz.tight_layout()
+
+
+###############################################################################
+# Fit a model to recover this receptive field
+# -------------------------------------------
+#
+# Finally, we'll use the :class:`mne.decoding.ReceptiveField` class to recover
+# the linear receptive field of this signal. Note that properties of the
+# receptive field (e.g. smoothness) will depend on the autocorrelation in the
+# inputs and outputs.
+
+# Create training and testing data
+train, test = np.arange(20), 21
+X_train, X_test, y_train, y_test = X[train], X[test], y[train], y[test]
+X_train, X_test, y_train, y_test = [np.rollaxis(ii, -1, 0) for ii in
+                                    (X_train, X_test, y_train, y_test)]
+# Model the simulated data as a function of the spectrogram input
+alphas = np.logspace(-4, 0, 10)
+scores = np.zeros_like(alphas)
+models = []
+for ii, alpha in enumerate(alphas):
+    mod = ReceptiveField(tmin, tmax, sfreq, freqs, model=Ridge(alpha))
+    mod.fit(X_train, y_train)
+
+    # Now make predictions about the model output, given input stimuli.
+    y_pred = mod.predict(X_test).squeeze()
+    scores[ii] = r2_score(y_pred, y_test[mod.mask_predict_])
+    models.append(mod)
+delay_sec_mod = mod.delays_ / sfreq
+
+# Choose the model that performed best on the held out data
+ix_best_alpha = np.argmax(scores)
+best_mod = models[ix_best_alpha]
+coefs = best_mod.coef_
+best_pred = best_mod.predict(X_test).squeeze()
+
+# Plot the original STRF, and the one that we recovered with modeling.
+fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(10, 5), sharey=True, sharex=True)
+ax1.pcolormesh(delays_sec, freqs, weights, cmap='RdBu_r')
+ax2.pcolormesh(delay_sec_mod, mod.feature_names, coefs, cmap='RdBu_r')
+ax1.set_title('Original STRF')
+ax2.set_title('Best Reconstructed STRF')
```

Comment:
https://3441-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_receptive_field.html

The zeros show up as blue, can you make it so zero is actually zero by setting the `clim` to be symmetric?